### PR TITLE
Chore/function test cases

### PR DESCRIPTION
**Pull request by tomchentw**
Hi thanks for creating this lib. I found the module currently did not work with components defined by function declaration. I have added a test case here for your reference. I have tried to update rust code but I'm not familiar enough with it, sorry!

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -58,6 +58,28 @@ mod test {
...
+            export const Component = function() { return <div />; }
+        "#,
+        /* Output */ r#"
+            export const Component = function() { return <div />; }
+            Component.displayName = "Component";
+        "#
+    );
+
+    test!(SYNTAX, runner,
+        /* Name */ fn_declaration_export,
```
Comment (tomchentw) [inline src/lib.rs:73]: Note: this will fail

**Comment by hjkcai** [comment]
Sorry for the late reply. I am not familiar with rust either😂 However I will try my best to implement this. Thanks!

**Comment by tomchentw** [comment]
No problem! This is definitely an awesome work

**Comment by hjkcai** [comment]
搞定~ Done! v0.2.0 published.

Note that you have to upgrade `@swc/core` to `^1.3.58`

**Comment by tomchentw** [comment]
Thank you! @hjkcai

**Pull request merged**